cmd/alertmanager/main.go

```diff
@@ -474,6 +477,13 @@ func run() int {
 		go disp.Run()
 		go inhibitor.Run()
 
+		err = tracingManager.ApplyConfig(conf)
```

Comment:
I'm just documenting this here as I'm not so familiar with this code, this is not feedback or a request for change. This code here is run when the `Coordinator`, which is responsible for reloading configurations at runtime, calls the event handlers registered via the `Subscribe` method. This specific function is responsible for reading the configuration and setting up the receivers, routes, inhibition rules, time intervals, inhibitor, silences, etc.